cmd/revokeactivation.go

```diff
@@ -227,7 +220,15 @@ func (p *RevokeActivationParams) Run(ctx ActionCtx) (store.Status, error) {
 	r := store.NewDetailedReport(true)
 	StoreAccountAndUpdateStatus(ctx, token, r)
 	if r.HasNoErrors() {
-		r.AddOK("revoked activation %s for account %s", p.export.Name, p.accountKey.publicKey)
+		if p.accountKey.publicKey == "*" {
```

Comment:
jwt.All?

Also, when adding * shouldn't you be able to clear all revocations prior to the provided time

Reply:
jwt lib should do the cleanup.